content/spin/v3/dynamic-configuration.md

```diff
@@ -260,9 +262,27 @@ container = "<cosmos-container>"
 
 > Note: The CosmosDB container must be created with the default partition key, `/id`.
 
-Whilst a single default store may be sufficient for certain application use cases, each Spin application can be configured to support multiple stores of any `type`, as shown in the `runtime-config.toml` file below:
+### AWS DynamoDB Key Value Store Provider
+
+To use an Amazon Web Services DynamoDB database as a backend for Spin's key/value store, change the type to `aws_dynamo` and specify your database account details:
+
+```toml
+[key_value_store.default]
+type = "aws_dynamo"
+region = "<aws-region>"   # e.g. "us-east-1"
+table = "<dynamo-table>"  # e.g. "spin-key-values"
+consistent_read = true    # optional, to use strongly consistent reads
+```
 
-> **Note:** At present, when deploying an application to Fermyon Cloud only the single "default" key-value store is supported. To see more about Spin support on Fermyon Cloud, visit the [limitations documentation](/cloud/faq#spin-limitations):
+You may optionally provide `access_key` and `secret_key` credentials; otherwise, Spin picks these up from your [AWS environment variables](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html). For short-lived credentials you can instead provide `token`.
```

Comment:
Apologies, the AWS credentials stuff is a bit funky. One must provide  `access_key` and `secret_key`, as well as potentially `token`. Just one small change

For short-lived credentials you can instead provide `token`.

to

For short-lived credentials you can additionally provide `token` from e.g. [AWS STS getSessionToken](https://docs.aws.amazon.com/STS/latest/APIReference/API_GetSessionToken.html)

Reply:
Aha, thanks.  I also added a link to the CLI wrapper.